docs/openapi/components/schemas/credentials/GS1DataCredential.yml

```diff
@@ -0,0 +1,126 @@
+        - VerifiableCredential
+        - GS1DataCredential
+  id:
+    description: A DID for this credential. The ID is not resolvable because the credential is sensitive information and only the user company can decide when it is presented.
+    type: string
+  name:
+    type: string
+  description:
+    type: string
+  issuanceDate:
+    type: string
+  expirationDate:
+    type: string
+  issuer:
+    description: The DID for the user company. It is not necessary that the DID be resolvable as the user company may wish to keep it private.
+    type: string
+  credentialSubject:
+    title: Subject Organization
+    description: Allocated a GS1 Member Organization.
+    type: object
+    properties:
+      sameAs:
+        description: If the credential subject ID is not a GS1 Digital Link URI, this specifies the GS1 Digital Link URI.
+        type: string
+      keyAuthorisation:
+        description: Reference to a credential that authorises the issuer to declare data for the credential subject (the key). Normally, this is the key credential itself, in which case the issuer of this and the key credential must be the same. For data that is declared by other parties on behalf of the issuer of the key credential, a delegation credential may be provided instead. In some circumstances, where the issuer of the data declaration is entirely independent, the key authorisation may be entirely outside of the GS1 system (e.g., a proof-of-purchase Verifiable Credential for a product review).
+        type: string
+      dataCertification:
+        description: List of references to credentials that certify that the issuer is competent to declare data of the given type. This would be used, for example, to declare that a solution provider is certified in accordance with the GS1 Package Measurement Rules Standard would assure the verifier of the credential that the planogram data contained in the credential is accurate. The combination of a data authorisation credential from the brand owner for planogram data along with the GS1 Package Measurement Rules Standard data certification credential would confirm to the verifier that the solution provider is both authorised and competent to provide the data. Multiple data certification credentials may be provided, which may address situations where multiple certifying parties exist and different users require different certifications. It’s not always necessary to have a data certification credential to match a key delegation credential, as data certification may be implied by the issuer of the credential (e.g., a Halal or other religious practice certification authority) that is trusted by the community that is interested in the data.
+        type: array
+        items:
+          type: string
+  credentialStatus:
+    type: object
+  proof:
+    $ref: ../snippets/proof.yml
+additionalProperties: true
+required:
+  - '@context'
+  - type
+  - issuanceDate
+  - issuer
+  - credentialSubject
+  - credentialStatus
```

Comment:
credentialStatus should not be required, If i wish to issue a non-status tracked credential the vocabulary shouldn't restrict me.

Reply:
Right! My bad.